### PR TITLE
Add set and delete access

### PR DESCRIPTION
**Pull request by Hellowlol**

### CODE REVIEW

**Review by fondberg — COMMENTED** [review]
Looks good but linting fails. Run black on the file as per the readme

**Comment by Hellowlol** [comment]
There is no mention about black in this projects readme only on https://github.com/fondberg/easee_hass#development. Ill add that to this project too and relint :)

**Comment by fondberg** [comment]
```
Development
This project uses black for code formatting and flake8 for linting. Two errors should be ignored when running flake8 so run it with

flake8 --ignore=E501,E231 custom_components/
```

File: easee/charger.py [hunk 1]
```diff
@@ -211,3 +211,19 @@ async def set_max_charger_current(self, current: int):
...
+                "open_for_all": 1,
+                "easee_account_required": 2,
+                "whitelist": 3
+        }
+
+        return await self.easee.put(f"/api/chargers/{self.id}/access", json=json[access])
+
+    async def delete_access(self):
+        """ Revert permissions overridden on a charger level"""
+        return await self.easee.delete(f"/api/chargers/{self.id}/access"
```
Comment (fondberg) [inline easee/charger.py:229]: missing `)` at end of line

**Comment by fondberg** [comment]
0.7.17 released

**Pull request merged**